packages/core/src/percy.js

```diff
@@ -142,7 +142,12 @@ export class Percy {
   }
 
   // Starts a local API server, a browser process, and internal queues.
-  async *start() {
+  async *start(execType) {
+    this.execType = execType;
```

Comment:
instead of setting this here, you can pass it to `throwIfTypeInvalid` function [ unless you are planning to use it in multiple locations ]